### DISCUSSION TITLE
Homebrew removing Jamf Pro binary

### DISCUSSION THREAD

**Discussion by JureJerebic** [Homebrew/discussions#4183] [Everyday usage]
### Output of `brew config`

```shell
Warning: Broken symlinks were found. Remove them with brew cleanup:
/usr/local/bin/jamf
```

### Output of `brew doctor`

```shell
Warning: Broken symlinks were found. Remove them with brew cleanup:
/usr/local/bin/jamf
```

### Description of issue

Hi,

It seems that using homebrew wants the users to remove the jamf binary. This is completely unacceptable and not only presents the possibility of breaking the connection with our MDM (Jamf Pro), but also triggers many security reports in our Jamf Protect environment, and reports this to the Information Security. 

Why does this happen, and is there a way to prevent brew from touching /usr/local/bin/jamf in the first place?

Thanks!

**Comment by SMillerDev** [comment]
> Why does this happen

Because brew expects to be the only user of it's prefix and because that symlink doesn't point anywhere.

> and is there a way to prevent brew from touching /usr/local/bin/jamf in the first place?

Install brew outside of `/usr/local` and waive all support options and precompiled binaries.

**Comment by carlocab** [comment]
What's the output of
```
file /usr/local/bin/jamf
file -L /usr/local/bin/jamf
```
?
